docs/README.md

```diff
@@ -26,7 +26,7 @@ source .venv/bin/activate
 python -m pip install -r requirements.txt
 python -m pip install -r ../pytorch-cpu-requirements.txt
```

Comment:
nit: requirements and pytorch-cpu-requirements are at the same dir level. We do not need to go one level up.

Reply:
The requirements.txt for docs is at the docs level and pytorch-cpu-requirements is one level up. This is not the same requirements.txt at the top level.